### PR TITLE
Edit Instructions Text To Make Goal Clearer

### PR DESCRIPTION
**Pull request by alypilkons**
The instructions end with "using the object provided" which is unclear since the user is expected to create their own name for the object. Users may think they need to reference an object name and be confused when they do not find it. Reiterating that the user needs to create the name for the object themselves makes the task clearer.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use--to-import-everything-from-a-file.english.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ You may use any name following the <code>import * as </code>portion of the state
 
 ## Instructions
 <section id='instructions'>
-The code in this file requires the contents of another file, <code>"capitalize_strings"</code>, that is in the same directory as the current file. Add the appropriate <code>import *</code> statement to the top of the file, using the object provided.
+The code in this file requires the contents of another file, <code>"capitalize_strings"</code>, that is in the same directory as the current file. Add the appropriate <code>import * as</code> statement to the top of the file. You can name the object that receives the imported values whatever you'd like.
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use--to-import-everything-from-a-file.english.md]: Is this necessary?  The paragraph above already has
> You may use any name following the `<code>import * as </code>`portion of the statement

Adding the 'as' is definitely useful, though. Could you add it at the start of the description, line 9, too?

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
Comments below!

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use--to-import-everything-from-a-file.english.md (hunk 1 above)
Comment (alypilkons) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use--to-import-everything-from-a-file.english.md]: It is a bit redundant. Removed that last sentence and added "as" to line 9.

Comment (ojeytonwilliams) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use--to-import-everything-from-a-file.english.md]: That looks a lot better without 'using the object provided'.  I suspect that line was relevant at some point, but now it's just confusing.

Thanks for sorting that out.

**Review by ojeytonwilliams — APPROVED** [review]

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use--to-import-everything-from-a-file.english.md (hunk 1 above)
Comment (alypilkons) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use--to-import-everything-from-a-file.english.md]: Thanks :)

**Review by thecodingaviator — APPROVED** [review]
LGTM

**Comment by thecodingaviator** [comment]
Thanks for contributing to the freeCodeCamp repository. Congratulations on your first PR! 🎊 

Look forward to more contributions from you! ✈️

**Comment by alypilkons** [comment]
thanks @thecodingaviator! 🎉

**Pull request merged**